src/test/java/academy/softserve/edu/elements/wrappers/AbstractClickableElement.java

```diff
@@ -0,0 +1,75 @@
+package academy.softserve.edu.elements.wrappers;
+
+import academy.softserve.edu.elements.interfaces.ILocator;
+import academy.softserve.edu.utils.Logger;
+import org.openqa.selenium.StaleElementReferenceException;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.interactions.Actions;
+import org.openqa.selenium.support.ui.ExpectedConditions;
+import org.openqa.selenium.support.ui.WebDriverWait;
+
+public abstract class AbstractClickableElement<T> extends AbstractElement {
+
+    private int timeout = 20;
+
+    AbstractClickableElement(final WebDriver driver, final ILocator locator) {
+        super(driver, locator);
+    }
+
+    final public T click() {
+
+        waitUntilElementIsPresent();
+        waitUntilElementIsClickable();
+        driver.findElement(locator.getBy()).click();
+        Logger.logInfo("<b><font size='1' color='black'>Clicked On "
```

Comment:
no need to make whole message bold, just locator name.
also do not configure font size